evol/population.py

```diff
@@ -78,6 +78,20 @@ def generate(cls, init_func, eval_func, size=100) -> 'Population':
         chromosomes = [init_func() for _ in range(size)]
         return cls(chromosomes=chromosomes, eval_function=eval_func)
 
+    @property
+    def individual_weights(self):
```

Comment:
do we want other people to use this property? i can't imagine it ... in which case id sooner put it in `Individual` but `_individual_weights` should also be fine.

Reply:
We can't put it in `Individual`, as it requires knowledge of the highest and lowest fitness. I'll make it private.